server/package.json

```diff
@@ -35,10 +35,8 @@
     "typescript": "5.0.4"
   },
   "scripts": {
-    "build": "yarn clean && tsc && yarn copy-assets && yarn create-commit-sha && yarn create-version-file",
+    "build": "yarn clean && tsc && yarn copy-assets",
     "copy-assets": "node ./bin/copy_server_assets.js",
-    "create-commit-sha": "node ./bin/create_commit_sha_file.js",
-    "create-version-file": "node ./bin/create_version_file.js",
```

Comment:
No more generating static files for this, this is computed in the config and served from memory when the endpoint `/commit` or `/version` are hit